src/edu/psu/sweng500/team8/play/GameSession.java

```diff
@@ -106,12 +123,23 @@ public void doRedo() {
 	public void enterNumber(CellCoordinates cellCoordinates, int number) {
 		
 		Cell currentCell = board.getCell(cellCoordinates.getRowIndex(), cellCoordinates.getColumnIndex());
-		SudokuAction sudokuAction = new SudokuAction(cellCoordinates, currentCell.getNumber(), number);
 		
-		/* TODO Actual Board Update to be implemented */
+		/* TODO - come back after PencilMarkMaanger is implement
+		 * current assumption is that PecilMarkManager will be a snapshot of the current pencil mark status 

```

Comment:
Ah -- that answers my question from earlier. The PencilMarkManager is a singleton (rather 1 per board) that controls pencil mark updates for the whole board. Each cell has a "getPencilMarks" method which is just a set of integers.